WBooks/Tabs/Library/LibraryViewModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LibraryViewModel.swift
+//  WBooks
+//
+//  Created by Megan Cole on 28/02/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class LibraryViewModel {
+    public let books: [BookViewModel] = [
```

Comment:
We use to make private array like this, to keep any modifications in one place... 
you can set this let like "private(set)" for make the setter private but the getter public
or you can set this "private" and make a function that returns the array